client/my-sites/plans/ecommerce-trial/upgrade-confirmation/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { useTranslate } from 'i18n-calypso';
+import Main from 'calypso/components/main';
+import './style.scss';
+import ConfirmationTask from './confirmation-task';
+import { getConfirmationTasks } from './confirmation-tasks';
+
+const TrialUpgradeConfirmation = () => {
+	const translate = useTranslate();
+
+	const tasks = getConfirmationTasks( { translate } );
+
+	return (
+		<Main wideLayout>
+			<div className="trial-upgrade-confirmation__header">
+				<h1 className="trial-upgrade-confirmation__title">
+					{ translate( 'Woo! Welcome to eCommerce' ) }
+				</h1>
+				<div className="trial-upgrade-confirmation__subtitle">
+					{ translate( 'Your purchase has been completed and you’re on the Commerce plan.' ) }
```

Comment:
I think we should use substitution for the plan name to start with.
```suggestion
					{ translate(
						'Your purchase has been completed and you’re on the %(planName)s plan.',
						{
							args: { planName: 'Commerce' } /* we should try to pull this from the plan object in calypso-products */
						}
					) }
```